Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/lists/List.tsx

```diff
@@ -49,6 +50,7 @@ const List = <TValue extends unknown>({
     wrap = false,
     fullWidth = false,
     scrollable = false,
+    itemsCount,
```

Comment:
Should have a default value or additional logic in order to handle if items count is not passed as an argument

Reply:
Done.

Reply:
How is this resolved?

Reply:
It is in commit 3091e03